tests/orchestrator/test_workflow_runner.py

```diff
@@ -259,8 +258,9 @@ def _setup_mock_workflow_in_service(request, inputs=None):
     workflow = models.Operation(
         name=mock_workflow_name,
         service=service,
-        implementation='workflow.mock_workflow',
-        inputs=inputs or {})
+        function='workflow.mock_workflow',
+        inputs=inputs or {},
+        arguments=inputs or {})
```

Comment:
No, not in the way this particular test works, since it creates an operation model directly and doesn't call `configure()`